components/eam/src/physics/cam/micro_p3_interface.F90

```diff
@@ -1024,6 +1029,8 @@ subroutine micro_p3_tend(state, ptend, dtime, pbuf)
     ite     = state%ncol
     kts     = 1
     kte     = pver
+
+!OG do we want dry or wet pressure here?
```

Comment:
I don't think that this is related to any bigger changes, because this variable I think is only used within P3's parameterizations. I looked into this and the answer seems to be, weirdly, that wet pressure would be correct here.

The reason is that, while we do want `pdeldry` to be passed in as `dpres` so that P3 is calculating densities in terms of dry air, `pres` is actually only used for two things, the calculation of water vapor diffusivity (`dv`) and as an argument for `qv_sat`. For `dv`, I don't think that that wet vs. dry matters, though my guess is that this formula comes from an empirical fit using measurements of total (wet) pressure. However, `qv_sat` is weird, because it requires a wet pressure as an input, but produces a dry mixing ratio as an output. You can see that [here](https://github.com/E3SM-Project/scream/blob/c814965fe67b7c307d5c39b615430781d1afca67/components/eam/src/physics/cam/wv_sat_scream.F90#L51), where the water vapor pressure is being subtracted from the input pressure (which only makes sense if the input is wet pressure). And the formula is correct if you are calculating dry saturation mixing ratio, from a wet pressure.

So wet pressure is good for both of the uses of `pres`, but it would be weird to pass both wet pressure and `pdeldry`. If this `qv_sat` is only used by P3, I would suggest that we modify it to accept dry rather than wet pressure, and then continue to pass only dry pressure variables to P3. At least, to me that feels like the least obnoxious solution.